sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs

```diff
@@ -102,21 +102,26 @@ internal class ChangeFeedFactory
                 return ChangeFeed.Empty();
             }
 
-            JsonDocument jsonMetaSegment;
-            if (async)
-            {
-                jsonMetaSegment = await JsonDocument.ParseAsync(
-                    blobDownloadInfo.Content,
-                    cancellationToken: cancellationToken
-                    ).ConfigureAwait(false);
+            JsonDocument jsonMetaSegment = null;
```

Comment:
Is there any downside to going with `using` here?

Reply:
Ah, I guess it is because of the branching that it is tricky.

Reply:
@JoshLove-msft  yeah. I suppose this could work?
```
using JsonDocument jsonMetaSegment = async ? 
	await JsonDocument.ParseAsync(
		 blobDownloadInfo.Content,
	         cancellationToken: cancellationToken).ConfigureAwait(false) :		
	JsonDocument.Parse(blobDownloadInfo.Content);
```

Reply:
nvm, seems to break a bunch of tests. Will leave it as `try-finally`